publication-model/src/main/java/no/unit/nva/model/instancetypes/degree/DegreePhd.java

```diff
@@ -20,6 +22,24 @@ public DegreePhd(@JsonProperty(PAGES_FIELD) MonographPages pages,
         this.related = related;
     }
 
+    @JacocoGenerated
+    @Override
+    public boolean equals(Object o) {
+        if (!(o instanceof DegreePhd degreePhd)) {
+            return false;
+        }
+        if (!super.equals(o)) {
+            return false;
+        }
+        return Objects.equals(getRelated(), degreePhd.getRelated());
+    }
+
+    @JacocoGenerated
+    @Override
+    public int hashCode() {
+        return Objects.hash(super.hashCode(), getRelated());
+    }
```

Comment:
Lack of equals method here made that relatedDocuments has never been verified in assertions in test.